machine_learning/losses/binary_cross_entropy.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
```

Comment:
Please include an explanation of BSE (along with a link to a source as well as the formula) in a header docstring